core/storage/user/gae_models.py

```diff
@@ -3011,3 +3011,89 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
         """
         empty_dict: Dict[str, base_models.EXPORT_POLICY] = {}
         return dict(super(cls, cls).get_export_policy(), **empty_dict)
+
+
+class LearnerGroupUserModel(base_models.BaseModel):
+    """Model for storing user's learner groups related data.
+
+    Instances of this class are keyed by the user id.
+    """
+
+    invited_to_learner_groups = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+    member_of_learner_groups = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+    progress_sharing_permissions = (
```

Comment:
Do the same variable rename here as well.

Reply:
Done!